### ISSUE TITLE
Missing alt text in images

### ISSUE THREAD

**Issue by LiteApplication** [lite-corp/nsi#2]
[These](https://github.com/lite-corp/website-nsi/blob/main/projets/index.html#L39-L53) images do not have alt text. This attribute is mandatory.
```html
<img src="../img/turtle.gif" alt="" class="projet_img"> 
<img src="../img/pendu.png" alt="" class="projet_img"> 
<img src="../img/allumettes.png" alt="" class="projet_img">
<img src="../img/bouledeneige.png" alt="" class="projet_img">
```
